core/src/main/java/org/web3j/ens/EnsResolver.java

```diff
@@ -376,13 +418,14 @@ private OffchainResolverContract lookupOffchainResolver(String ensName) throws E
                 getResolverAddress(ensName), web3j, transactionManager, new DefaultGasProvider());
     }
 
-    private String getResolverAddress(String ensName) throws Exception {
-        NetVersion netVersion = web3j.netVersion().send();
-        String registryContract = Contracts.resolveRegistryContract(netVersion.getNetVersion());
-
-        ENS ensRegistry =
-                ENS.load(registryContract, web3j, transactionManager, new DefaultGasProvider());
+    private OffchainResolverContract lookupOffchainResolver(String ensName, Credentials credentials)
+            throws Exception {
```

Comment:
it's recommended to throw specific exception and avoid the Exception class for better track, only and advice for future

Reply:
Not able to throw particular Exception as - 
```
public T send() throws Exception {
        return callable.call();
    }
```

If I set it to IOException, getting error - Unhandled exception: java.lang.Exception
  org.web3j.protocol.core.RemoteCall<T> 
  public T send()
throws Exception